cdist/conf/type/__pacman_conf/manifest

```diff
@@ -0,0 +1,111 @@
+#get params
```

Comment:
Please ensure there is a GPLv3 header in all files - as all files in cdist are GPLv3